### ISSUE TITLE
LaTeX variable with underscore not preserved

### ISSUE THREAD

**Issue by ledjohnny** [augustt198/latex2sympy#12]
If I run:
`print(process_sympy("B_{\\nu}"))`
I get:
`B`

Can support be added for outputting `B_{\\nu}`?

Awesome script BTW!

**Comment by augustt198** [comment]
Yeah that's definitely a problem. Something like `x_0 + x_1` shouldn't equal `2x`.

Do you have any ideas about what symbol name would be returned by `B_{\\nu}`? Maybe `B_nu`...

**Comment by ledjohnny** [comment]
Yeah, I suppose the output should be `B_nu`.

**Comment by sindytn** [comment]
I gave this a shot for subscripted functions and variables. Take a look at my pull request (#16). Thanks!
